frontend/components/FieldAsset/FieldAsset.css

```diff
@@ -0,0 +1,28 @@
+.container {
+  width: 100%;
+  margin-top: 9px;
+  height: 40px;
+  line-height: 25px;
+  font-size: 15px;
+  box-sizing: border-box;
+  border: 1px solid #e4e4e4;
+  border-radius: 5px;
+  outline: 0;
+  display: flex;
+  flex-wrap: wrap;
+  align-items: center;
+}
+
+.container[preview] {
```

Comment:
You're adding a non-standard attribute to the element, in the actual rendered DOM, without it even being prefixed with `data-`. There's no reason to do this when all you want to do is target it for styling.

That isn't the case in `FieldString`, but I can change it to also use a class if that makes you happier.